.changeset/healthy-cherries-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/react': patch
+---
+
+Event map type leak resolved. Component will build without type errors when no event map is provided.
```

Comment:
I'd phrase this in a way that makes sense to someone reading a changelog without all the context:

"Fixed an error with typings when no event map is provided to `createComponent()`. Instance properties were being confused with event handlers.

Reply:
sounds good

Reply:
done